client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -262,6 +266,23 @@ def serialize_model(model):
         return bytestream, method, model_type
 
     # `model` is an instance
+    pyspark_ml_base = maybe_dependency("pyspark.ml.base")
+    if pyspark_ml_base:
+        # https://spark.apache.org/docs/latest/api/python/_modules/pyspark/ml/base.html
+        pyspark_base_classes = (
+            pyspark_ml_base.Estimator,
+            pyspark_ml_base.Model,
+            pyspark_ml_base.Transformer,
+        )
+        if isinstance(model, pyspark_base_classes):
+            temp_dir = tempfile.mkdtemp()
+            try:
+                spark_model_dir = os.path.join(temp_dir, "spark-model")
+                model.save(spark_model_dir)
+                bytestream = zip_dir(spark_model_dir)
+            finally:
+                shutil.rmtree(temp_dir)
+            return bytestream, "zip", "pyspark"
```

Comment:
Model API and Model service (as I understand it) check for a "serialization method" and a "model type", so this PR introduces "zip" and "pyspark", respectively.

Reply:
I _think_ we need more info than `pyspark` if we want to deserialize, but we can handle that later. let's just add a TODO for now